apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -128,6 +134,21 @@ type Resources struct {
 	Total *corev1.ResourceRequirements `json:"total,omitempty"`
 }
 
+type SearchSpec struct {
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Max search time range allowed"
+	MaxSearchDuration string `json:"maxSearchDuration,omitempty"`
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Limit used for search requests if none is set by the caller"
+	DefaultResultLimit int `json:"defaultResultLimit,omitempty"`
```

Comment:
What is the default value? 

What is the search request?  A request to the object store?

Reply:
The default value is the default value of tempo configuration (which in this case is 20) https://grafana.com/docs/tempo/latest/configuration/#query-frontend